### ISSUE TITLE
[Feature Request] Order User Video Activity by updated date

### ISSUE THREAD

**Issue by Norwelian** [streamaserver/streama#808]
TL;DR: I want a button or something that enables you to see the User Video Activity entries ordered by updated date, not just by creation date.

Long explanation:
Through extensive use on the part of my friends of my personal streama, one of the things I love the most is watching on the "Admin>User Activity>Video Activity" tab their use of the webpage.
But, strangely enough, I've been noticing that some of the videos they watch (I know this because of server logs that I have on the reverse proxy) were not saved on the list.
Just now I came to the realization that they were being saved, but as they had an entry already on the DB, they were not showing up on the front, they were just being updated on their current position.
So, after this long and stupid explanation in order to explain myself better, the request is having a button or something that enables you to see the User Video Activity entries ordered by updated date, not just by creation date.

**Comment by Ircas** [comment]
I would love to see this. 
There should be more buttons to actually sort maybe by time as @norwelian suggested maybe sort by video and ip.

Streama is getting better with every update...

**Comment by Norwelian** [comment]
I temporarily changed this in my personal streama.
Found how to do this work, but without a button or something to turn it on/off, now it always shows the entries by lastUpdated date.
While we wait for a proper solution (one that I'm unable to do at the time), I'll put here my changes if anyone wants the same thing as me meanwhile (the user filter still works):
In "grails-app/assets/javascripts/streama/templates/settings-user-activity.tpl.htm", line 69:
Before:
`      <tr ng-repeat="activity in vm.userActivity.list">`
After:
`      <tr ng-repeat="activity in vm.userActivity.list | orderBy: '-lastUpdated'">`

**Comment by dularion** [comment]
added for next release
